common/presentation/src/main/kotlin/com/pandulapeter/myTodos/presentation/ui/Demo1.kt

```diff
@@ -42,13 +43,15 @@ internal fun ColumnScope.Demo1(
 }
 
 @Composable
-private fun ColumnScope.SourceSelector(
+private fun SourceSelector(
     shouldUseRemoteSource: Boolean,
-    onShouldUseRemoteSourceChanged: (Boolean) -> Unit
+    onShouldUseRemoteSourceChanged: (Boolean) -> Unit,
 ) = Row(
     modifier = Modifier
-        .align(Alignment.CenterHorizontally)
+        .fillMaxWidth()
         .padding(top = Dimension.contentPaddingExtraLarge),
+    horizontalArrangement = Arrangement.Center,
+    verticalAlignment = Alignment.CenterVertically
```

Comment:
The same alignment can be achieved like this instead of making the children extension functions on `ColumnScope`